docker-app/qfieldcloud/core/models.py

```diff
@@ -1003,6 +1003,24 @@ def get_project_file_storage_default() -> str:
     return settings.STORAGES_PROJECT_DEFAULT_STORAGE
 
 
+def get_project_attachments_file_storage_default() -> str:
+    """Get the default attachments file storage for the newly created project.
+
+    Returns:
+        the name of the storage
+    """
+    return settings.STORAGES_PROJECT_ATTACHMENTS_DEFAULT_STORAGE
+
+
+def get_project_are_attachments_versioned_default() -> bool:
+    """Get the default value for versioning of project attachments.
+
+    Returns:
+        whether the attachments are versioned by default
+    """
+    return settings.STORAGES_PROJECT_ATTACHMENTS_ARE_VERSIONED_DEFAULT
```

Comment:
The setting is called `STORAGE_PROJECT_DEFAULT_ATTACHMENTS_VERSIONED` (not `...ARE_VERSIONED_DEFAULT`)